server/routes/api/user.js

```diff
@@ -1,7 +1,52 @@
 var router = require('express').Router(),
+    validator = require('validator'),
     Responses = require('../../responses/api'),
     User = require('../../db/model/User.js');
 
+router.post('/profile', function(req, res) {
```

Comment:
I'd rather do a post for this just because the type of data. I'm open to changing it though